packages/kit/src/views/Market/MarketDetailV2/components/TokenActivityOverview/utils/formatTokenActivityData.ts

```
@@ -3,40 +3,51 @@ import type { IMarketTokenDetail } from '@onekeyhq/shared/types/marketV2';
+        totalVolume = Number(tokenDetail.volume8h) || buyVolume + sellVolume;
         break;
       case '24h':
         buys = Number(tokenDetail.buy24hCount) || 0;
         sells = Number(tokenDetail.sell24hCount) || 0;
-        buyVolume = Number(tokenDetail.volumeBuy24h) || 0;
-        sellVolume = Number(tokenDetail.volumeSell24h) || 0;
+        buyVolume = Number(tokenDetail.vBuy24h) || 0;
+        sellVolume = Number(tokenDetail.vSell24h) || 0;
+        totalVolume = Number(tokenDetail.volume24h) || buyVolume + sellVolume;
         break;
       case '1h':
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)